### PR TITLE
docs: updated commenting and external links

### PR DESCRIPTION
**Pull request by tdivoll**
Some cleanup based on project meeting today. MVP to get the new page live and then focus on updates with current feedback and any that comes in after going live.

### CODE REVIEW

**Comment by tdivoll** [comment]
I updated the easy things and put the more complicated ones as issues for later after the page is launched.

File: src/components/buoy-telemetry/BuoyMap.vue [hunk 1]
```diff
@@ -44,12 +44,25 @@ const props = defineProps<{
...
+  chlorophyllSizeRange: [0.2, 0.65],
+};
+const colorRange = {
+  chlorophyllColorRange: ["#93C572", "#32CD32", "#355E3B"],
+};
+
 const annotatedSamples = computed(() => {
-  const domain = [0, 45];
-  const sqrtScale = scaleSqrt().domain(domain).range([0.2, 0.65]);
+  const domain = varDomain.chlorophyllDomain;
```
Comment (eldu) [inline src/components/buoy-telemetry/BuoyMap.vue:59]: then here you could deconstruct like this

```js
const { chlorophyll: {varDomain, sizeRange, colorRange} = config
```

File: src/components/base/ExternalLink.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (eldu) [inline src/components/base/ExternalLink.vue:1]: OOh thanks for adding in the external link component! What do you think of it?

**Review by eldu — APPROVED** [review]
approving here to move this forward. I did suggest another clean up. looks good to me though!

**Review by mirestrepo — APPROVED** [review]
Agree with Ellen :) if you can, push the changes to the `config` object before merging

File: src/components/base/ExternalLink.vue (hunk 2 above)
Comment (tdivoll) [reply src/components/base/ExternalLink.vue:1]: I like it! And it makes it more flexible to use social media icons if we get to the point that we need to link those kinds of things.

File: src/components/buoy-telemetry/BuoyMap.vue (hunk 1 above)
Comment (tdivoll) [reply src/components/buoy-telemetry/BuoyMap.vue:59]: In typescript, I think something is missing. I get this error and I'm not sure what `(property)` should be? 
`(property) chlorophyll?: {
    chlorophyll: {
        varDomain: number[];
        markerSize: number[];
        markerColors: string[];
    };
}
A destructuring declaration must have an initializer.ts(1182)`

**Pull request merged**